### PR TITLE
PDB Merge

### LINKED ISSUE #15: pdb_merge
From the discussion in #13, `pdb_merge.py` should merge multiple PDB's chain information into a single file.  I understand that `TER` lines that do not exist in the original file should not be added (this is a job for `pdb_tidy.py`). But, should `pdb_merge.py` consider the original `TER` statements if they exist or only the `ATOM` and `HETATM` lines?

#discussion

### PR DESCRIPTION
**Pull request by joaomcteixeira**
Merges multiple PDB files into a single PDB file. `TER` lines will be lost. Accepts piping.

Addresses #13 and Closes #15

### CODE REVIEW

File: pdb_merge.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    return args
+
+def _concatenate_pdbs(pdbfnlh):
+    """
+    Concatenates ATOM, HETATM and TER lines from multiple PDBs.
+    
+    Parameters:
+        - pdbfnlh (list of str): list of file names to process.
+    """
+    # reads only ATOM, HETATOM and TER lines
```
Comment (JoaoRodrigues) [inline pdb_merge.py]: Comment is not entirely accurate

File: pdb_merge.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import os
+import sys
+import re
+
+__author__ = "Joao M.C. Teixeira"
+__email__ = "joaomcteixeira@gmail.com"
+
+USAGE = __doc__.format(__author__, __email__)
+
+def check_input(args):
```
Comment (JoaoRodrigues) [inline pdb_merge.py]: I would collapse the not len() and else statements in one, to keep it simpler to read:
```
if len(args) > 1:
    # do stuff
else:
    print(usage)
    sys.exit(1)
```

**Comment by joaomcteixeira** [comment]
solved

**Comment by JoaoRodrigues** [comment]
Merged and integrated in version 2. Thanks!

**Pull request closed without merging**